components/serializer.rst

```diff
@@ -755,7 +755,7 @@ Here, we set it to 2 for the ``$child`` property:
 The metadata loader corresponding to the chosen format must be configured in
 order to use this feature. It is done automatically when using the Symfony
 Standard Edition. When using the standalone component, refer to
-:ref:`the groups documentation <component-serializer-attributes-groups>` to
+:ref:`the groups documentation <attributes-groups>` to
```

Comment:
There is a `span` element with this id in the docs. But it seems that some JavaScript or the like is breaking the handling of anchors on the documentation website currently. `attributes-groups` anchor that you used here is generated automatically based on the headline. However, reStructuredText does not allow to just use this anchor. To be able to use it in a `ref` role one must define an explicit label.